core/dirty_check.py

```
@@ -179,14 +179,24 @@ async def check(*text: Union[str, List[str]], msg: Bot.MessageSession = None, ad
 
 async def check_bool(*text: Union[str, List[str]]) -> bool:
+    '''检查字符串是否合规。
+
+    :param text: 字符串（List/Union）。
+    :returns: 字符串是否合规。
+    '''
     chk = await check(*text)
     for x in chk:
         if not x['status']:
             return True
     return False
```

Ruff (0.8.0)

Use `return any(not x['status'] for x in chk)` instead of `for` loop

Replace with `return any(not x['status'] for x in chk)`

(SIM110)